slim.Dockerfile

```
@@ -0,0 +1,23 @@
+FROM python:3.7-slim as runtime
+EXPOSE 8080
+
+WORKDIR /girder
+
+RUN apt-get update \
+  && apt-get install -qy --no-install-recommends \
+    git \
+    libldap2-dev \
+    libsasl2-dev \
+  && rm -rf /var/lib/apt/lists/*
+COPY . /girder/
+
+# avoid psutil GCC dependency by using unofficial large_image wheel
+RUN pip install \
+  --no-cache-dir \
+  --upgrade \
+  --find-links https://girder.github.io/large_image_wheels \
+  --upgrade-strategy eager .
```

Since we're installing into a fresh image, there won't be any existing packages to `--upgrade` anyway. You should be able to omit:
```suggestion
  --find-links https://girder.github.io/large_image_wheels \
  .
```

`pip`, `setuptools`, and `wheel` are pre-installed, but since they're not in the transitive dependency tree, they won't be affected by any attempts to upgrade.